lib/view_component/preview.rb

```diff
@@ -17,6 +17,15 @@ def render(component, **args, &block)
       }
     end
 
+    def render_component(component, &block)
```

Comment:
Should this accept `args`?

Reply:
Yes, otherwise [this feature](https://github.com/github/view_component#:~:text=It's%20also%20possible%20to%20set%20dynamic%20values%20from%20the%20params%20by%20setting%20them%20as%20arguments) won't work